packages/code-du-travail-modeles/src/modeles/base/indemnite-licenciement.yaml

```
@@ -122,6 +122,30 @@ contrat salarié . indemnité de licenciement . facteur plus de dix ans cc:
       facteur: 1 / 3
   unité: €
 
+contrat salarié . indemnité de licenciement . résultat identique au légal moins de 10ans:
```

Je porposerai un nommage plus explicite:

```suggestion
contrat salarié . indemnité de licenciement . résultat conventionnel identique au légal moins de 10ans:
```